src/models/user.model.js

```diff
@@ -11,7 +11,6 @@ const User = sequelize.define('users', {
   username: {
     type: Sequelize.STRING,
     allowNull: false,
-    unique: true,
```

Comment:
why removing username unique as true, where as in team we have concluded that it is going to continue to be unique, and there are some places we are  destroying user's token based on username. this is done in most test cases dne before this PR, so I think you have to let it as it was. because we can not find a user from DB, where his username is the same like others. check here: 
<img width="917" alt="image" src="https://user-images.githubusercontent.com/91186046/228773911-7ab43a94-560a-4ee0-83cb-044fde83a903.png">